packages/workflows/__tests__/api/Properties.unit.test.ts

```diff
@@ -713,7 +869,21 @@ describe("Get workflow properties", () => {
             expectZosmfResponseSucceeded(response, error);
             expect(response).toEqual(PRETEND_ZOSMF_RESPONSE);
         });
+        it("Flatten the substeps", async () => {
+
+            let error: ImperativeError;
+            let response: any;
+            try {
+                response = await PropertiesWorkflow.processStepSummaries(STEP_WITH_SUBSTEPS);
+                Imperative.console.info(`Response ${response}`);
+            } catch (thrownError) {
+                error = thrownError;
+                Imperative.console.info(`Error ${error}`);
+            }
 
+            expectZosmfResponseSucceeded(response, error);
+            expect(response).toEqual(FLATTEN_STEP);
```

Comment:
Totally up to you, but you could expect to match a snapshot. This way you wouldn't need to maintain the validation structure.

Reply:
Thanks @tucker01 , I have been having some issues with the build (I know why they are but I will not get into details here) so I decided to make it like this.